Formula/zimpl.rb

```diff
@@ -0,0 +1,25 @@
+class Zimpl < Formula
+  desc "Language to translate the mathematical model of a problem into a MI(N)LP"
+  homepage "https://zimpl.zib.de"
+  url "https://scipopt.org/download/release/scipoptsuite-8.0.3.tgz"
+  version "3.5.3"
+  sha256 "5ad50eb42254c825d96f5747d8f3568dcbff0284dfbd1a727910c5a7c2899091"
+  license "LGPL-3.0-only"
+
+  depends_on "cmake" => :build
+  depends_on "bison"
```

Comment:
can't use the one from macOS?

Reply:
Unfortunately, not. Bison at least 2.4.0 is required.